src/FakeItEasy/EventAction.cs

```diff
@@ -0,0 +1,91 @@
+namespace FakeItEasy
+{
+    using FakeItEasy.Core;
+
+    /// <summary>
+    /// Represents subscription to or unsubscription from an event of a fake.
+    /// </summary>
+    /// <remarks>An instance of this class can't be explicitly created. Use the static <c>Add</c> and <c>Remove</c> methods to obtain an instance.</remarks>
+    public abstract class EventAction

```

Comment:
I realized belatedly that `EventAccessor` would probably be a better name for this. If you agree, I'll rename.

Reply:
I hadn't minded `EventAction`. I'm not sure `EventAccessor` feels better to me, but no worse, so if you prefer, I'm fine.

While reading, I was mulling over alternatives. I wonder:

```c#
A.CallTo(fake, EventSubscription.Remove())
```
? I'm not entirely sold, but present it in the interest of confusion.